platforms/jvm/testing-jvm/src/integTest/groovy/org/gradle/testing/TestTaskJvmArgsProviderIntegrationTest.groovy

```diff
@@ -59,9 +59,9 @@ class TestTaskJvmArgsProviderIntegrationTest extends AbstractIntegrationSpec {
                 }
             }
 
-            test.jvmArgumentProviders << new MyTestSystemProperties(
+            test.jvmArgumentProviders.add(new MyTestSystemProperties(
                 inputFile: file(providers.gradleProperty('inputFile').get())
-            )
+            ))
```

Comment:
❔ How hard would it be to support `<<` for Groovy only for multi-valued properties? If not a lot, I think it would be useful, even if we deprecated it straight away.

Reply:
This should be addressed with https://github.com/gradle/gradle/issues/23637